lib/tmuxinator/project.rb

```diff
@@ -135,7 +135,8 @@ def pre_window
       elsif pre_tab?
         yaml["pre_tab"]
       else
-        yaml["pre_window"]
+        pre_window = yaml["pre_window"]
+        pre_window.is_a?(Array) ? pre_window.join("; ") : pre_window
```

Comment:
Since this will be the third time this logic is used in this file, we should probably break it out into a private method and throw it at the bottom of the class somewhere.